NomaiGrandPrix/SpawnActionFactory.cs

```diff
@@ -57,5 +61,53 @@ private static void LowerZeroGElevator()
             elevator._targetLocalPos = elevator._endLocalPos;
             elevator.StartLift();
         }
+
+        private static void CreateAshTwinCampfire()
+        {
+            var timeLoopRing = GameObject.Find("TimeLoopRing_Body/Interactibles_TimeLoopRing_Hidden");
+            var campfirePrefab = Locator._hourglassTwinA._rootSector.transform.Find(
+                "Sector_NorthHemisphere/Sector_NorthSurface/Sector_Lakebed/Interactables_Lakebed/Lakebed_VisibleFrom_Far/Prefab_HEA_Campfire"
+            );
+            var campfire = GameObject.Instantiate(campfirePrefab.gameObject, timeLoopRing.transform);
+
+            campfire.transform.localPosition = new Vector3(18, 0.18f, -21);
+            campfire.transform.Rotate(new Vector3(95, 0, 38), Space.Self);
```

Comment:
would be good to explicitly mark these as float literals with a trailing f

Reply:
additionally it might be a good idea to pull these constants out into a const variable?

Reply:
done